common/containers/App/index.jsx

```diff
@@ -87,24 +72,18 @@ function mapStateToProps(state) {
         nodeToggle: state.config.nodeToggle,
         languageSelection: state.config.languageSelection,
         languageToggle: state.config.languageToggle
-    }
+    };
 }
 
 function mapDispatchToProps(dispatch) {
     return {
         changeNode: (i: number) => {
-            dispatch(CHANGE_NODE(i))
-        },
-        toggleNodeDropdown: () => {
-            dispatch(TOGGLE_NODE_DROPDOWN())
+            dispatch(CHANGE_NODE(i));
```

Comment:
Is this right? Thought we were passing the whole object now? should be changed to `i: any`?